### DISCUSSION TITLE
Is it still maintaining?

### DISCUSSION THREAD

**Discussion by kondohiroki** [hyn/multi-tenant#1022] [Q&A] [answered]
I'm using this package for my project and it's on production now (still working)
I just wondering because it's been a long time from the last updated or I missing something?

**Comment by bkintanar** [comment, marked as answer]
@kondohiroki yes this is still being maintained. This is actually a very mature package now and we're doing bug fixes and security fixes for this package only. We also have a more robust package https://github.com/tenancy/tenancy if you want to check that out.

**Reply by kondohiroki** [reply]
Sound nice!, Lovely
Thank you very much
